lib/content-services/src/lib/category/categories-management/categories-management.component.html

```diff
@@ -13,41 +19,16 @@
             <button
                 data-automation-id="categories-remove-category-button"
                 mat-icon-button
-                [class.adf-btn-padded]="!isCRUDMode"
                 (click)="removeCategory(category)"
                 [attr.title]="removeCategoryTitle | translate"
                 [disabled]="disableRemoval">
                 <mat-icon>remove</mat-icon>
             </button>
         </span>
     </div>
-    <div *ngIf="((!categoryNameControlVisible && categories.length)) || categoryNameControlVisible"
-        [hidden]="!categoryNameControlVisible"
-        class="adf-category-name-field">
-        <mat-form-field>
-            <mat-icon matPrefix>search</mat-icon>
-            <mat-label id="adf-category-name-input-label">
-                {{ 'CATEGORIES_MANAGEMENT.NAME' | translate }}
-            </mat-label>
-            <input
-                #categoryNameInput
-                matInput
-                autocomplete="off"
-                [formControl]="categoryNameControl"
-                (keyup.enter)="addCategory()"
-                aria-labelledby="adf-category-name-input-label"
-                adf-auto-focus
-            />
-            <mat-error [hidden]="!categoryNameControl.invalid">{{ categoryNameErrorMessageKey | translate }}</mat-error>
-        </mat-form-field>
-        <button
-            mat-icon-button
-            [class.adf-btn-padded]="!isCRUDMode"
-            (click)="hideNameInput()"
-            [attr.title]="'CATEGORIES_MANAGEMENT.HIDE_INPUT' | translate">
```

Comment:
CATEGORIES_MANAGEMENT.HIDE_INPUT - is that used somewhere? I can't see, so probably we can remove that key from en.json